spec/functional/resource/yum_package_spec.rb

```diff
@@ -308,84 +308,95 @@ def pkg_arch
     context "with constraints" do
       it "with nothing installed, it installs the latest version" do
         flush_cache
-        yum_package.package_name("chef_rpm >= 1.2")
+        yum_package.package_name("chef_rpm")
+        yum_package.version("1.10-1")
```

Comment:
yeah this is not equivalent, and these specs going red means that this change actually breaks this API.